clustered/ehcache-clustered/build.gradle

```diff
@@ -107,7 +107,7 @@ tasks.named('jar') {
   osgi {
     instruction Constants.BUNDLE_SYMBOLICNAME, 'org.ehcache.clustered'
     instruction Constants.EXPORT_PACKAGE, '!com.tc.*, !com.terracotta.*, !org.terracotta.*, !org.ehcache.*.internal.*, !sun.misc, org.ehcache.clustered.client.*, org.ehcache.clustered.common.*'
-    instruction Constants.IMPORT_PACKAGE, '!sun.misc.*, org.ehcache.xml.*;resolution:=optional, jdk.jfr.*;resolution:=optional, !com.fasterxml.jackson.*, !org.terracotta.json, javax.xml.bind*;version="[2.2,3)", *'
+    instruction Constants.IMPORT_PACKAGE, '!sun.misc.*, org.ehcache.xml.*;resolution:=optional, jdk.jfr.*;resolution:=optional, !com.google.gson.*, !org.terracotta.json, javax.xml.bind*;version="[2.2,3)", *'
```

Comment:
It would be nice to eliminate this "naturally" by not having JSON related stuff on the classpath that gets noticed by BND... out of scope for this PR though (and probably a pipe dream).

Reply:
so update to how it should be... soon ;-)